geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateGatewaySenderCommand.java

```diff
@@ -88,12 +97,16 @@ public ResultModel createGatewaySender(@CliOption(key = {CliStrings.GROUP, CliSt
           help = CliStrings.CREATE_GATEWAYSENDER__BATCHTIMEINTERVAL__HELP) Integer batchTimeInterval,
 
       @CliOption(key = CliStrings.CREATE_GATEWAYSENDER__ENABLEPERSISTENCE,
+          specifiedDefaultValue = "true",
+          unspecifiedDefaultValue = "false",
           help = CliStrings.CREATE_GATEWAYSENDER__ENABLEPERSISTENCE__HELP) Boolean enablePersistence,
 
       @CliOption(key = CliStrings.CREATE_GATEWAYSENDER__DISKSTORENAME,
           help = CliStrings.CREATE_GATEWAYSENDER__DISKSTORENAME__HELP) String diskStoreName,
 
       @CliOption(key = CliStrings.CREATE_GATEWAYSENDER__DISKSYNCHRONOUS,
+          specifiedDefaultValue = "true",
+          unspecifiedDefaultValue = "true",
```

Comment:
you sure this is the unspecified default value?

Reply:
Hey @jinmeiliao, yes I'm sure:
https://github.com/apache/geode/blob/8ee587c0d2e301775dae6a1eecba4f9a9a258ca6/geode-core/src/main/java/org/apache/geode/internal/cache/wan/GatewaySenderAttributes.java#L68
https://github.com/apache/geode/blob/8ee587c0d2e301775dae6a1eecba4f9a9a258ca6/geode-core/src/main/java/org/apache/geode/cache/wan/GatewaySender.java#L37-L40